### PR TITLE
save wallet name & url

### PR DESCRIPTION
**Pull request by Bosun-Josh121**

### CODE REVIEW

File: src/utils/chromeStorageHelpers.ts [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import { ChromeStorageObject } from '../services/types/chromeStorage.types';
+
+export const saveAccountDataToChromeStorage = async (
+  chromeStorageService: any, // Accept as an argument
```
Comment (danwag06) [inline src/utils/chromeStorageHelpers.ts]: Use `ChromeStorageService` type here

File: src/utils/chromeStorageHelpers.ts [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  if (!account) return;
+
+  // Define the key for storage
+  const key: keyof ChromeStorageObject = 'accounts';
+
+  // Prepare the update for the account's name and icon
+  const update: Partial<ChromeStorageObject['accounts']> = {
+    [account?.addresses.identityAddress]: {
+      ...account,
+      name: enteredAccountName,
```
Comment (danwag06) [inline src/utils/chromeStorageHelpers.ts]: use defaults here:
```
name: enteredAccountName || account.name,
icon: enteredAccountIcon || HOSTED_YOURS_IMAGE,
```

**Review by danwag06 — CHANGES_REQUESTED** [review]
This looks good after the requested changes have been addressed. You can also clean up unnecessary code comments.

**Comment by danwag06** [comment]
This looks good @Josh-121 will get it merged at the completion of the hackathon. Nice work.

**Comment by Bosun-Josh121** [comment]
> This looks good @Josh-121 will get it merged at the completion of the hackathon. Nice work.

Thanks 🙏

**Review by danwag06 — APPROVED** [review]

**Pull request merged**